cmd/cli/namespace_test.go

```diff
@@ -409,8 +409,8 @@ var _ = Describe("Running the namespace list command", func() {
 		)
 
 		// helper function that adds a name space to the clientset
-		addNamespace := func(name, mesh string) {
-			ns := createNamespaceSpec(name, mesh, false)
+		addNamespace := func(name, mesh string, enableSideCarInjection bool) {
```

Comment:
Please add this test to `mesh_list_test.go` and not `namespace_test.go`.

Reply:
I do not understand what do you mean.
I modified the osm namespace list, not the osm mesh list.
Can you elaborate more?

Reply:
My bad, looks fine!